browse/pathmap.py

```diff
@@ -0,0 +1,29 @@
+"""Makes GCP path map string
+
+For use with gcloud compute url-maps add-path-matcher
+
+See https://cloud.google.com/load-balancing/docs/url-map
+"""
+import sys
+
+from browse.factory import create_web_app
+
+
+backend_service = sys.argv[1]
+
+app = create_web_app()
+
+
+def mappingline(rule):
+    hasParam = '<' in rule.rule
+    if hasParam:
+        path = rule.rule.split('<')[0] + "*"
+    else:
+        path = rule.rule
+
+    return f"{path}={backend_service}"
+
+
+with app.app_context():
+    lines = [mappingline(path) for path in app.url_map.iter_rules()]
+    print(','.join(lines))
```

Comment:
Mostly informational comment. I ran this from the command line and got:
```
/stats/main=phoenix.arxiv.org,/tb/recent=phoenix.arxiv.org,/category_taxonomy=phoenix.arxiv.org,/prevnext=phoenix.arxiv.org,/index=phoenix.arxiv.org,/form=phoenix.arxiv.org,/abs=phoenix.arxiv.org,/ct=phoenix.arxiv.org,/=phoenix.arxiv.org,/static/browse/0.3.2.6/*=phoenix.arxiv.org,/static/base/0.16.9/*=phoenix.arxiv.org,/auth/show-endorsers/*=phoenix.arxiv.org,/src/*=phoenix.arxiv.org,/src/*=phoenix.arxiv.org,/tb/redirect/*=phoenix.arxiv.org,/tb/redirect/=phoenix.arxiv.org,/show-email/*=phoenix.arxiv.org,/trackback/*=phoenix.arxiv.org,/trackback/=phoenix.arxiv.org,/archive/*=phoenix.arxiv.org,/e-print/*=phoenix.arxiv.org,/archive/*=phoenix.arxiv.org,/cookies/*=phoenix.arxiv.org,/archive/=phoenix.arxiv.org,/cookies=phoenix.arxiv.org,/format/*=phoenix.arxiv.org,/bibtex/*=phoenix.arxiv.org,/stats/*=phoenix.arxiv.org,/year/*=phoenix.arxiv.org,/year/*=phoenix.arxiv.org,/list/*=phoenix.arxiv.org,/html/*=phoenix.arxiv.org,/year/*=phoenix.arxiv.org,/year/*=phoenix.arxiv.org,/refs/*=phoenix.arxiv.org,/cits/*=phoenix.arxiv.org,/list/=phoenix.arxiv.org,/list=phoenix.arxiv.org,/pdf/*=phoenix.arxiv.org,/div/*=phoenix.arxiv.org,/abs/*=phoenix.arxiv.org,/abs/=phoenix.arxiv.org,/ps/*=phoenix.arxiv.org,/tb/*=phoenix.arxiv.org,/tb/=phoenix.arxiv.org
```
It looks like it's capturing both routes for the home page, i.e. `/=phoenix.arxiv.org` and `/index=phoenix.arxiv.org`.

Reply:
Should this file live at the top level of the project, since it's importing from `browse.factory`?

Reply:
It doesn't need to go at the top level due to that import.